### PR TITLE
clean up docs and tutorials, add setup guide

### PR DESCRIPTION
**Pull request by slmjkdbtl**
- added `doc/setup.md` for a setup guide
- added `doc/dev.md` for dev guide (wip)
- cleaned up some markdown related css
- added the dark mode switch on markdown pages
- moved `sprites/`, `sounds/`, and `fonts/` under `assets/`
- renamed `tut/` to `doc/`
- added tutorial entries to `/doc` page

### CODE REVIEW

File: doc/dev.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Developing Kaboom
```
Comment (masad-frost) [inline doc/dev.md:1]: ty

Comment (masad-frost) [reply doc/dev.md:1]: Usually these docs are called `Contributing.md`

File: doc/setup.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+and it'll find the Kaboom package and include it in the built `build.js`, include `build.js` in your HTML and you're good to go. Feel free to automate this process.
+
+## Loading Assets
+
+You might have encountered errors when trying to `loadSprite()` from local file system, that is because browser won't allow loading local files with JavaScript. To get around that you'll need to use a static file that serves the files through HTTP. There're a lot of programs that helps you to do that.
+
+- `$ python3 -m http.server` if you have [python3](https://www.python.org) installed
+- `$ python -m SimpleHTTPServer` if you have [python2](https://www.python.org) installed
+- `$ serve` if you have [serve](https://github.com/vercel/serve) installed
+- `$ caddy file-server` if you have [caddy](https://caddyserver.com/) installed
```
Comment (masad-frost) [inline doc/setup.md:73]: What about https://github.com/amasad/static-here :P

**Review by masad-frost — APPROVED** [review]
Noice

File: doc/setup.md (hunk 2 above)
Comment (slmjkdbtl) [reply doc/setup.md:73]: hh yep!

**Pull request merged**